addons/test_mail/tests/test_mail_resend.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import itertools
+
+from odoo.addons.test_mail.tests import common
+from odoo.tests import tagged
+from odoo.tools import mute_logger
+from odoo.tools.safe_eval import safe_eval
+from odoo import api
+
+
+@tagged('resend_test')
+class TestMailResend(common.BaseFunctionalTest, common.MockEmails):
+
+    @classmethod
+    def setUpClass(cls):

```

Comment:
Please reuse existing data from BaseFunctionalTest and TestRecipients it will lessen the diff